src/parser/ast_common.c

```diff
@@ -50,12 +51,8 @@ AST_NodeEntity* New_AST_NodeEntity(char *alias, char *label, Vector *properties)
 	ne->t = N_ENTITY;
 	ne->properties = properties;
 	
-	if(alias != NULL) {
-		ne->alias = strdup(alias);
-	}
-	if(label != NULL) {
-		ne->label = strdup(label);
-	}
+	if(alias != NULL) ne->alias = alias;
```

Comment:
if no duplication is required then we don't need to condition this assignment.

Reply:
:+1: